src/Plugin/PreviewLinkAutopopulate/Guides.php

```diff
@@ -36,13 +38,16 @@ public function getPreviewEntities(): array {
     elseif ($node->bundle() == 'localgov_guides_page') {
       $overview = $node->get('localgov_guides_parent')->entity;
     }
-    $guide_nodes[] = $overview;
 
-    // Find guide pages.
-    $guide_pages = $overview->get('localgov_guides_pages')->referencedEntities();
-    foreach ($guide_pages as $guide_page) {
-      if ($guide_page instanceof NodeInterface && $guide_page->access('view')) {
-        $guide_nodes[] = $guide_page;
+    if ($overview instanceof Node) {
```

Comment:
Nitpik: this should be NodeInterface I think.

Reply:
And one more @millnut  :)

Reply:
👍 @ekes I've now updated this